src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -597,6 +597,9 @@ static guint16 sri_vector_methods [] = {
 	SN_Max,
 	SN_Min,
 	SN_Multiply,
+	SN_Negate,
+	SN_Not,
```

Comment:
I don't think there is a method `Not` for `Vector64`

Reply:
Changed it.